### DISCUSSION TITLE
请问 umi4 的 targets 相关配置在哪里看（详细的）？现在想要兼容 webview 某些版本怎么设置呢？

### DISCUSSION THREAD

**Discussion by yuanliu147** [umijs/umi#11658] [Q&A]
请问 umi4 的 targets 相关配置在哪里看（详细的）？现在想要兼容 webview 某些版本怎么设置呢？

**Comment by fz6m** [comment]
umi 是一个 react 面向中后台的 SPA 应用框架，故默认面向的用户对象是兼容到现代浏览器，不建议将 umi 使用到有更高兼容要求和手机端的场景。

当你需要适当降低 umi 的兼容浏览器版本时，可以在 [`targets`](https://umijs.org/docs/api/config#targets) 里配置 `chrome` / `firefox` / `edge` / `safari` 这几个浏览器相应的版本，对于历史项目可参考 [非现代浏览器兼容](https://umijs.org/blog/legacy-browser) 勉强兼容。
